### DISCUSSION TITLE
SDXL multiple concepts in one lora

### DISCUSSION THREAD

**Discussion by meanin2** [TheLastBen/fast-stable-diffusion#2588] [Q&A]
Anyone know how to train multiple concepts into a single lora for sdxl? (runpod)

I was thinking to train one concept and then once that's trained, continue training in the session with the next concept with new images.
Would that work? 
Any better methods?

Also is there a way to train off a different SDXL base model?

**Comment by TheLastBen** [comment]
Train all the concepts at the same time, rename the images accordingly, like concept_a(1).png, concept_a(2).png ..... concept_b(1).png, concept_b(2).png ... etc
Training the lora on the base model gives the best possible output, training it on a custom model will make the lora give good results only on that custom model

**Reply by meanin2** [reply]
Thank you!
What if my training data set for each concept is 15 images or more?
Would I need to adjust the LR or epochs?

**Reply by TheLastBen** [reply]
adjust the settings depending on the total number of images, the bigger the number, the lower the learning rate and the epochs, to avoid overtraining
